screens/FindGamePage.js

```diff
@@ -1,44 +1,72 @@
 import React from 'react';
-import { StyleSheet, Text, View, Image, FlatList, TouchableOpacity } from 'react-native';
+import { StyleSheet, Text, View, Image, FlatList, TouchableOpacity, Dimensions } from 'react-native';
 import { Font } from 'expo';
 import { StackNavigator } from 'react-navigation';
 
 export class FindScreen extends React.Component {
   static navigationOptions = { header: null };
   state = {
     fontLoaded: false,
+    pressed: [{
```

Comment:
You should have changed this to be `years: []` and `places: []` instead of a long list